build_daemon/pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: build_daemon
-version:  4.0.0
+version:  4.0.1-dev
 description: A daemon for running Dart builds.
 repository: https://github.com/dart-lang/build/tree/master/build_daemon
 
 environment:
-  sdk: ">=2.14.0 <3.0.0"
```

Comment:
Should we consider removing the use of `unawaited`, publishing, then adding it bak with the constraint bump?

There could be folks with a 2.14.0 SDK who are broken right?

Reply:
If we don't get reports of an issue, I'd say leave it. All of our stats show folks move pretty quickly to our latest SDKs.

Reply:
Realistically they couldn't get a version solve under 2.17 anyways, so it doesn't matter.